libraries/chain/include/eosio/chain/block_header_state.hpp

```diff
@@ -27,17 +63,18 @@ struct block_header_state {
     vector<uint8_t>                   confirm_count;
     vector<header_confirmation>       confirmations;
 
-    block_header_state   next( const signed_block_header& h, bool trust = false )const;
-    block_header_state   generate_next( block_timestamp_type when )const;
+   pending_block_header_state  next( block_timestamp_type when, uint16_t num_prev_blocks_to_confirm )const;
+
+   block_header_state   next( const signed_block_header& h, bool skip_validate_signee = false )const;
 
-    void set_new_producers( producer_schedule_type next_pending );
-    void set_confirmed( uint16_t num_prev_blocks );
-    void add_confirmation( const header_confirmation& c );
-    bool maybe_promote_pending();
+    //void set_new_producers( producer_schedule_type next_pending );
+    //void set_confirmed( uint16_t num_prev_blocks );
+    //void add_confirmation( const header_confirmation& c );
+    //bool maybe_promote_pending();
```

Comment:
Remove commented out code. There is also some in the source file that could also be removed.

Reply:
Resolved in https://github.com/EOSIO/eos/pull/6588/commits/2cb4dbc8825af74f45c18f5b35d030d2888bfd3c.